### DISCUSSION TITLE
BUG: most options doesn't work in a cell of a table

### DISCUSSION THREAD

**Discussion by ronaldvanvugt** [docmost/docmost#424] [Help / Questions]
**Expected behavior:** if I use an option in a cell of a table (like Bullet list, code or image), this options is applied to the cell

**Actual behavior:** I can select the option in a cell of a table, but the option isn't applied

**How to replicate:** Try to add a Bullet list in a cell of a table

By the way, I love Docmost!

**Comment by luciana-rodrigues** [comment]
Hello! I discovered Docmost recently and I'm already in love with it. I missed this same behavior as well, hope they implement it as the software grows.

**Comment by mustafa-yilmaz** [comment]
According to the review shared (by @Philipinho) [here](https://github.com/docmost/docmost/pull/679#issuecomment-2621923321) at https://github.com/docmost/docmost/pull/679, those kinds of additions are not on the short-term development agenda of Docmost .
